woocommerce/woocommerce-ios
# Update Alamofire to v5

## Description

This PR updates Alamofire to the latest version, because WordPressKit and WordPressAuthenticator no longer depend on Alamofire.

**I highly recommend reviewing this PR commit by commit.** The Alamofire API migration happens in their own commits.

## Testing instructions

Testing the app using both a site that's connected to WP.com and self-hosted site that is not connected to Jetpack.

Verify a few different app features, for example:
1. Login/Logout
2. Add/remove product
3. Upload images to product

## Screenshots

N/A

---
